src/redux/sagas/motions/moveFundsMotion.ts

```diff
@@ -112,65 +97,197 @@ function* moveFundsMotion({
         'annotateMoveFundsMotion',
       ]);
 
-    const [fromPermissionDomainId, fromChildSkillIndex] = yield call(
-      getPermissionProofs,
-      colonyClient.networkClient,
-      colonyClient,
-      fromDomain.nativeId,
-      [ColonyRole.Funding],
-      votingReputationClient.address,
-    );
-
-    const [, toChildSkillIndex] = yield call(
-      getPermissionProofs,
-      colonyClient.networkClient,
-      colonyClient,
-      toDomain.nativeId,
-      [ColonyRole.Funding],
-      votingReputationClient.address,
-    );
-
     const isOldVersion = colonyVersion <= 6;
     const contractMethod = isOldVersion
-      ? `moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)`
+      ? 'moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)'
       : 'moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,uint256,uint256,address)';
 
-    const encodedAction = colonyClient.interface.encodeFunctionData(
-      contractMethod,
-      [
-        ...(isOldVersion ? [] : [fromPermissionDomainId, constants.MaxUint256]),
-        fromPermissionDomainId,
-        fromChildSkillIndex,
-        toChildSkillIndex,
-        fromPot,
-        toPot,
-        amount,
-        tokenAddress,
-      ],
-    );
+    // eslint-disable-next-line no-inner-declarations
+    function* getCreateMotionParams() {
+      const { nativeFundingPotId: fromPot } = fromDomain;
+      const { nativeFundingPotId: toPot } = toDomain;
 
-    // create transactions
-    yield fork(createTransaction, createMotion.id, {
-      context: ClientType.VotingReputationClient,
-      methodName: 'createMotion',
-      identifier: colonyAddress,
-      params: [
-        Id.RootDomain,
-        motionChildSkillIndex,
+      if (isMultiSig) {
+        const requiredRoles = REQUIRED_MULTISIG_ROLES_BY_OPERATION.moveFunds;
```

Comment:
I think you can get this directly from `PERMISSIONS_NEEDED_FOR_ACTION` and we should use `REQUIRED_MULTISIG_ROLES_BY_OPERATION` strictly for multi-sigs using the root motion saga.